### PR TITLE
Fixed test_endswith tests (passing wrong variable name)

### PR DESCRIPTION
**Pull request by henriquebraga**
Fixed test_endswith tests (variable 'end' wasn't setted properly). Also, was created a new test case with an empty string.

### CODE REVIEW

**Comment by henriquebraga** [comment]
I think something went wrong with my rebase :disappointed: . Going fix it.

(Sorry for the very delay)

**Comment by glasnt** [comment]
Hi @henriquebraga! If you need help resyncing your fork: https://help.github.com/articles/syncing-a-fork/

Let us know if you need a hand with the rebase 👍

**Comment by henriquebraga** [comment]
I think now it worked!

Really, I can't say thank you enough! <3

**Comment by glasnt** [comment]
Looks like you've got some trailing whitespace there. 

```
The command "py.test tests" exited with 0.
3.21s$ flake8
./tests/datatypes/test_str.py:18:38: W291 trailing whitespace
./tests/datatypes/test_str.py:24:38: W291 trailing whitespace
./tests/datatypes/test_str.py:34:50: W291 trailing whitespace
./tests/datatypes/test_str.py:40:38: W291 trailing whitespace
```

File: tests/datatypes/test_str.py [hunk 1]
```diff
@@ -14,25 +14,31 @@ def test_setattr(self):
...
-        #     print('abracadabra'.endswith('abra'))
-        #     """)
+        self.assertCodeExecution("""
+            print('abracadabra'.endswith('abra'))          
+            """)
+
+        self.assertCodeExecution("""
+            s = "ABRACADABRA"
+            suffix = ""
+            print(s.endswith(suffix))
```
Comment (glasnt) [inline tests/datatypes/test_str.py]: Just the trailing whitespace here, and then you're good to go 👍

Comment (henriquebraga) [inline tests/datatypes/test_str.py]: Thanks for the review and patience again! @glasnt :smile:

**Comment by glasnt** [comment]
✨
